packages/lib/src/spatial-navigation/components/FocusableView.tsx

```diff
@@ -82,7 +82,10 @@ type InnerFocusableViewProps = FocusableViewProps & {
 
 const InnerFocusableView = forwardRef<View, InnerFocusableViewProps>(
   ({ viewProps, webProps, children, isActive, isFocused, style }, ref) => {
-    const accessibilityProps = useSpatialNavigatorFocusableAccessibilityProps();
+    const accessibilityProps = {
+      ...useSpatialNavigatorFocusableAccessibilityProps(),
+      accessibilityState: { selected: isFocused },
+    };
```

Comment:
don't do it like this, it won't be memoized ;)

in that regard, we should maybe include it in useSpatialNavigatorFocusableAccessibilityProps (that does memoize it) but that might break things, safest would be to leave it here in a separate memo

```tsx
const accessibilityState = useMemo(() => ({ selected: isFocused }), [isFocused])
```

Reply:
Done :)